core/src/main/java/bio/terra/pearl/core/dao/BaseJdbiDao.java

```diff
@@ -257,6 +251,19 @@ public Optional<T> findByProperty(String columnName, Object columnValue) {
         );
     }
 
+    public List<T> findByTriggerId(UUID triggerId) {
```

Comment:
I don't think you need this method (and it doesn't belong in BaseJDbiDao regardless)

Reply:
Omg I totally fat fingered the Intellij ctrl-click and ended up in the base dao haha